akkaclusterback/src/main/resources/application.conf

```diff
@@ -19,10 +19,11 @@ akka {
 
   cluster {
     auto-down-unreachable-after = 10s
-    min-nr-of-members = 2
+//    min-nr-of-members = 2
     role {
-      frontend.min-nr-of-members = 1
-      backend.min-nr-of-members = 1
+   //   frontend.min-nr-of-members = 1
+   //   backend.min-nr-of-members = 1
```

Comment:
The problem with requiring a min number of members is that ConductR won't get a chance to hear from the service; at least with the existing approach of signalling ConductR re. readiness when the cluster is up. Remember that only one member will come up at a time...